src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.cpp

```diff
@@ -267,65 +223,59 @@ void TypeLookupManager::remove_remote_endpoints(
     }
 }
 
-StatefulWriter* TypeLookupManager::get_builtin_request_writer()
+fastrtps::rtps::SampleIdentity TypeLookupManager::get_type_dependencies(
+        const fastrtps::types::TypeIdentifierSeq& id_seq) const
```

Comment:
I suggest updating the API to start using already the new API `xtypes::TypeIdentifierSeq`

Reply:
Updated in [7dafd25](https://github.com/eProsima/Fast-DDS/pull/4126/commits/7dafd25b2d97a27f2f53acf522b0d2c68e72a46e)